pyOCD/target/target_kinetis.py

```diff
@@ -141,3 +164,20 @@ def massErase(self):
         else:
             logging.error("Failed to unlock. MDM status: 0x%x", val)
             return False
+
+    ## @brief Write the unsecure FSEC value to flash.
+    #
+    # This will prevent the device from being secured after the next reset.
+    def unlockFlash(self):
+        logging.info("Unlocking chip...")
+
+        # Init flash algorithm.
+        self.flash.init()
+
+        logging.info("Performing mass erase")
+        self.massErase()
```

Comment:
In AN4835 - "Production Flash Programming Best Practices for Kinetis K- and L-series MCUs" in 4.2.1 - "Mass erase," it states that as one of the steps of mass erase is, "5. The security byte in the flash configuration field is also programmed to 0xFE (unsecure). On subsequent resets, the processor will boot in unsecure mode unless the flash configuration field is modified."

This is also consistent with the behavior I have seen.  On flash init setTargetState is called, which calls resetStopOnReset.  This would re-lock the processor if mass erase didn't write to the FSEC register.  I verified that Kinetis devices can be reliably unlocked with the script blank_test.py.